tests/integration/rpc/backend/test_call_tx.go

```diff
@@ -292,8 +293,7 @@ func (s *TestSuite) TestSendRawTransaction() {
 	ethTx, bz := s.buildEthereumTx()
 
 	emptyEvmChainIDTx := s.buildEthereumTxWithChainID(nil)
-	invalidEvmChainIDTx := s.buildEthereumTxWithChainID(big.NewInt(1))
-
+	invalidChainID := big.NewInt(1)
```

Comment:
seems already included in current main https://github.com/cosmos/evm/blob/82cf029c0ddc6a89a5cce64c464e04312325a294/tests/integration/rpc/backend/test_call_tx.go#L296